src/main/java/org/broadinstitute/dsm/route/EventTypeRoute.java

```diff
@@ -22,7 +22,7 @@ public Object processRequest(Request request, Response response, String userId)
         String userIdRequest = UserUtil.getUserId(request);
         if (StringUtils.isNotBlank(realm)) {
             if (UserUtil.checkUserAccess(realm, userId, "participant_event", userIdRequest)) {
-                return EventType.getEventTypes(realm);
+                return EventTypeDao.getEventTypeByInstanceName(realm);
```

Comment:
having static methods make easier to use them directly but in general, it breaks object oriented programming idea.